### DISCUSSION TITLE
[@types/node] Typo in process.d.ts commentaries

### DISCUSSION THREAD

**Discussion by Shassk** [DefinitelyTyped/DefinitelyTyped#66627] [Issues with a @types package] [answered]
Specifically, a typo in this line

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/process.d.ts#L1278

where instead of this:
```
* If Node.js was not spawned with an IPC channel, `process.send` will be`undefined`.
```
should be this (a missed spacebar before `undefined`):
```
* If Node.js was not spawned with an IPC channel, `process.send` will be `undefined`.
```

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
I am on it

**Reply by Semigradsky** [reply, marked as answer]
Fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66634
